### PR TITLE
Another lib update break fix

### PR DESCRIPTION
**Pull request by sheetalkamat**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/dom-webcodecs/webcodecs.generated.d.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ interface VideoEncoderConfig {
     alpha?: AlphaOption | undefined;
     avc?: AvcEncoderConfig | undefined;
     bitrate?: number | undefined;
-    bitrateMode?: BitrateMode | undefined;
+    bitrateMode?: VideoEncoderBitrateMode | undefined;
```
Comment (jakebailey) [inline types/dom-webcodecs/webcodecs.generated.d.ts:140]: I was going to say that you can just revert the BitrateMode change from https://github.com/DefinitelyTyped/DefinitelyTyped/commit/7e99574c3ba0a7e13c1c8718971da60f408bbe97 but it looks like Nathan bumped it to TS 5.1+ anyhow, so, I guess this is fine.

**Review by jakebailey — APPROVED** [review]

File: types/dom-webcodecs/webcodecs.generated.d.ts (hunk 1 above)
Comment (jakebailey) [reply types/dom-webcodecs/webcodecs.generated.d.ts:140]: Not sure what criteria we use to distinguish when we should bump the minimum, versus copying and using typesVersions...

Comment (sheetalkamat) [reply types/dom-webcodecs/webcodecs.generated.d.ts:140]: I dont know either.

**Comment by sheetalkamat** [comment]
Ready to merge

**Pull request merged**